src/components/SignInForm/SignInForm.jsx

```diff
@@ -0,0 +1,124 @@
+import { useContext, useState } from "react";
+import styles from "./SignInForm.module.scss";
+import { AuthContext } from "../../containers/AuthContext";
+
+const SignInForm = () => {
+  const authContext = useContext(AuthContext);
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+  const [isVisibleMessage, setIsVisibleMessage] = useState(false);
```

Comment:
so, as i can see, you are using one state variable as a marker for "has Errors".
not the best approach, please check PR for SCRUM-37, SignUpBuer component - there is errors state in it.
validate input with `onBlur` event.

Reply:
Done